src/databases/models/User.js

```diff
@@ -62,7 +62,8 @@ module.exports = (sequelize, DataTypes) => {
       profile_pic_public_id: {type: DataTypes.STRING, allowNull: true},
       status: {type: DataTypes.BOOLEAN, allowNull: false},
       bio: {type: DataTypes.TEXT, allowNull: true},
-      blocked_by_admin: {type: DataTypes.BOOLEAN, allowNull: true},
+      blocked_by_admin: {type: DataTypes.BOOLEAN, defaultValue: 0, allowNull: true},
+      w_pu_user: {type: DataTypes.DOUBLE, defaultValue: 1, allowNull: true},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `blocked_by_admin` and `w_pu_user` fields are set to allow null values. However, you have provided default values for these fields. It's better to set `allowNull: false` to ensure that these fields always have a value and prevent potential issues with null values in the future.
```diff
-       blocked_by_admin: {type: DataTypes.BOOLEAN, defaultValue: 0, allowNull: true},
-       w_pu_user: {type: DataTypes.DOUBLE, defaultValue: 1, allowNull: true},
+       blocked_by_admin: {type: DataTypes.BOOLEAN, defaultValue: 0, allowNull: false},
+       w_pu_user: {type: DataTypes.DOUBLE, defaultValue: 1, allowNull: false},
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->